test/pages/add_task_test.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:task_list_app/cubit/task_cubit.dart';
+import 'package:task_list_app/models/task.dart';
+import 'package:task_list_app/pages/add_task.dart';
+import '../materializer.dart';
+import '../mocks.dart';
+
+void main() {
+  final MockGoRouter mockGoRouter = MockGoRouter();
+
+  group('Add Task Screen', () {
+    Task task = const Task(id: null, name: 'example task');
+    // Non-`go_router` test
+    testWidgets('Add Task title is displayed', (WidgetTester tester) async {
+      // Set up mock cubit(s) - This can be done in the setUp() if it's common to a group() of tests
+      final MockTaskCubit mockTaskCubit = MockTaskCubit();
+      when(() => mockTaskCubit.state).thenReturn(
+        const TaskState(tasks: []),
+      );
+      when(() => mockTaskCubit.readTasks()).thenAnswer((_) => Future.value());
+
+      // Render the widget in the file name
+      await tester.pumpWidget(Materializer(
+        mockCubits: [mockTaskCubit],
+        child: const AddTaskScreen(),
+      ));
+
+      final titleFinder = find.text('Add a Task');
+      expect(titleFinder, findsOneWidget);
+    });
+    testWidgets(
```

Comment:
By convention, I would expect a line break in between each test. Please add them for readability.

Reply:
done